src/bin/mina-indexer.rs

```diff
@@ -39,25 +36,14 @@ enum IndexerCommand {
     },
 }
 
-#[derive(Subcommand, Debug)]
-enum ServerCommand {
-    /// Start the mina indexer with a config file
-    Config {
```

Comment:
Here you are removing functionality that was indeed tested (to some minimal degree). Do we support a config file or no? If no, then surely there is some sort of YAML parsing code that should be deleted. If yes, then how do we specify where the config file is located?

Reply:
The functionality is not easily replicated. There was a reason why the additional CLI command was used in favour of command line/config file precedence. For sake of change latency I was going to create another issue to add it back in.

Reply:
Okay, if you create that issue, refer to it here, and then merge in current 'main', I'll approve and merge.